salt/top.sls

```diff
@@ -13,6 +13,7 @@ base:
     - etcd-proxy
     - flannel
     - docker
+    - container-feeder
```

Comment:
I think `container-feeder` is needed only on the minions (BTW we should really change the name to worker, but not right now...). Am I wrong?